README.rst

```diff
@@ -38,74 +38,149 @@ This repository uses `git-flow <https://github.com/nvie/gitflow>`_ conventions;
 contains the most recent release, and work in progress will be on the **develop** branch.
 Pull requests should be made against **develop**.
 
-
-Development instructions
-------------------------
-
-Bare-metal
-~~~~~~~~~~
-
-Initial setup and installation:
-
-- Recommended: create and activate a python 3.9 virtualenv::
-
-    virtualenv cdhweb -p python3.9
-    source cdhweb/bin/activate
-
-- Use pip to install required python dependencies.  To install production
-  dependencies only::
-
-    pip install -r requirements.txt
-
-  To install all development requirements::
-
-    pip install -r requirements/dev.txt
-
-- Install django-compressor dependencies::
-
-    npm install
-
-- Copy sample local settings and configure for your environment::
-
-    cp cdhweb/settings/local_settings.py.sample cdhweb/settings/local_settings.py
-
-You must add a ``SECRET_KEY`` value in your local settings.
-
-- Download licensed fonts and install locally under /sitemedia/fonts/
-
-- Install OpenCV dependencies (if necessary) for `wagtail image feature detection <https://docs.wagtail.io/en/stable/advanced_topics/images/feature_detection.html>`_
-
-Docker (Springload)
+-----------------
+Development Setup
+-----------------
+
+Choose one of the following setup options:
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Option 1: Local Development ("bare-metal")
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This option runs the application directly on your local machine.
+
+**Setup Steps:**
+
+1. **Set up Python environment:**
+   .. code-block:: bash
```

Comment:
This code block isn't rendering properly, probably because this file is ReStructured Text rather than Markdown for historical reasons (we include the readme in our sphinx docs for the project and was probably set up before there was markdown support for sphinx).